app/workers/update_rating_frequency_worker.rb

```diff
@@ -0,0 +1,10 @@
+class UpdateRatingFrequencyWorker
+  include Sidekiq::Worker
+
+  def perform(class_name, class_id, query)
+    class_ = class_name.safe_constantize
+    class_obj = class_.find(class_id)
+    class_.where(id: class_id).update_all(query)
```

Comment:
Is there a reason for this roundabout approach?  Couldn't you just call `class_obj.update(query)`?

Reply:
Well for some reason, running `Sidekiq::Testing.inline!`, converts the class into a hash in rspec, so a way around that was doing this